polling_stations/templates/base.html

```diff
@@ -46,8 +46,8 @@
         {% endblock site_meta %} 
 
     </head>
-
-    <body {% block body_attrs %}{% endblock body_attrs %} class="ds-scope">
+    {% block body_base %}
```

Comment:
I don't think we need to bring this back to just to use it in one place `base_embed.html` file. I think the aim should be work with the new `base.html` file. So I would try removing this again and then see comment about the `base_embed.html` structure